packages/ui/package.js

```diff
@@ -2,7 +2,8 @@ Package.describe({
   name: 'ui',
   summary: "Deprecated: Use the 'blaze' package",
   version: '1.0.13',
```

Comment:
We should probably bump the version here and publish the package so that the warning gets out.